src/components/cards/descriptionCards.js

```diff
@@ -0,0 +1,8 @@
+
+const CardDesc=(props)=>{
+    return(
+        <p>{props.cards.desc}</p>
```

Comment:
cards is plural word that implies that it is array, but it is not. Please use singular form for it